src/main/resources/api/openapi.yaml

```diff
@@ -1245,3 +1279,71 @@ paths:
             text/plain; charset=UTF-8:
               schema:
                 type: string
+
+  /wallet/assets/issue/generate:
+    post:
+      summary: Generate asset issue transaction
+      operationId: walletAssetIssueTransactionGenerate
+      tags:
+      - wallet
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/AssetIssueRequest'
+      responses:
+        '200':
+          description: Generated Ergo transaction
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErgoTransaction'
+        '400':
+          description: Bad asset issue request
+          content:
+            text/plain; charset=UTF-8:
+              schema:
+                type: string
+        default:
+          description: Error
+          content:
+            text/plain; charset=UTF-8:
+              schema:
+                type: string
+
+  /wallet/assets/issue:
+    post:
+      summary: Generate and send a transaction from payment
```

Comment:
should be something like `asset issue transaction`

Reply:
It is supposed that each of these routes relates to transaction creation - the differences are the action that will be taken after (send transaction or not) and the actual type of transaction ("asset issue" or "payment")